.github/workflows/markdown-links-verifier.yml

```diff
@@ -0,0 +1,14 @@
+name: Markdown links verifier
+on: [push, pull_request]
+
+jobs:
+  validate_links:
+    name: Markdown links verifier
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout the repository
+      uses: actions/checkout@v1
+
+    - name: Validate links
+      uses: Youssef1313/markdown-links-verifier@v0.0.7
```

Comment:
You're reviewing very old code. Ctrl+F5 your page.

Reply:
For the code that is currently there, it is using a different GitHub action that I explained in the comment above to Youssef. I also have a thread with the .NET OSS Admins about it via email. For now, it is our own discretion on GitHub Actions we use, but they recommend using commit SHA instead of version numbers (which I've done in the updated code).